package.json

```diff
@@ -97,7 +97,7 @@
     "ember-load-initializers": "^2.1.1",
     "ember-maybe-import-regenerator": "^0.1.6",
     "ember-qunit": "^4.6.0",
-    "ember-ref-modifier": "^1.0.1",
+    "ember-ref-bucket": "^1.0.0",
```

Comment:
This can be removed, as we already have it in `dependencies`. I realize we had also `ember-ref-modifier` in both places, which did not make really sense...